shell/components/ChartReadme.vue

```diff
@@ -17,34 +18,74 @@ export default {
       default: true
     }
   },
+  created() {
+    // When rendered as a standalone page, the theme is passed as a query parameter and applied to the body.
+    if (this.isStandalone) {
+      const theme = this.$route.query.theme || 'light';
+
+      document.body.classList.add(`theme-${ theme }`);
```

Comment:
I think it might make sense to create a separate template which includes the theme so you don't need to do this in the component.